### ISSUE TITLE
Motherboard Crash & My Return

### ISSUE THREAD

**Issue by MichaelTunnell** [MichaelTunnell/las-picks#21]
TL:DR
Crap happened that took me off development, I've built a temporary workaround to get back to work but it means I am currently not working off the Git source. Git will be out of date until I finish the current update.

-----

So I had a massive hardware failure last month...ON MY BIRTHDAY! and it took down my main workstation. I had an old laptop I could use that was good enough for doing some stuff but it was so slow that I couldn't stand it anymore. I bought a new laptop (well used) while waiting for my motherboard to be RMA'd. 

In the meantime, I was having issues with my new system as far as logging into some servers. This was a weird thing I won't go into but I think it was an error on my distro. I decided to give up on fixing it and just made a Virtual Machine with an Ubuntu based so I could get some reliability. I am now back up and running with the VM so I can get back to updating the LAS Picks site.

I finally got my motherboard back last week so I am excited to get that backup but because it took so long I am stuck using this laptop until I finish some jobs for a couple clients. My workflow is all kinds of screwed up right now. :)

Anyway, I am back in a temporary basis of updating the site. I have merged a pull request from crossroads and I am now working on filling the rest of the missing data. I still have a TON of work to do on the "archive data" but after today the current data will be back up to date. :)

**Comment by MichaelTunnell** [comment]
I've finally fixed my main machine but there are so many hundreds of updates for it (Arch with over 400 updates over 4 months) that I really just need to rebuild the system.

Anyway, I have completed the archiving of the data.php and I have pushed everything to this repo so everything is back up to date with over 2,100+ modifications since my last git push. :)
